src/Razor/src/rzls/Endpoints/UpdateLogLevelEndpoint.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts;
+using Microsoft.CodeAnalysis.Razor.Logging;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Endpoints;
+
+internal class UpdateLogLevelEndpoint(LogLevelProvider logLevelProvider) : IRazorNotificationHandler<UpdateLogLevelParams>
```

Comment:
An attribute for what endpoint this handles? And add it to DI?

Reply:
Yea, missed that locally while testing too :)